resolwe_bio/descriptors/unified-descriptor.yml

```diff
@@ -814,6 +814,326 @@
+          choices:
+            - label: MET
+              value: met
+        - name: mutation_status_ros1
+          label: Mutation status/ROS1
+          type: basic:string
+          required: false
+          choices:
+            - label: ROS1
+              value: ros1
+        - name: mutation_status_cdkn2a
+          label: Mutation status/CDKN2A
+          type: basic:string
+          required: false
+          choices:
+            - label: 'CDKN2A R80*'
+              value: 'cdkn2a_r80*'
+            - label: 'CDKN2A R58*'
+              value: 'cdkn2a_r58*'
+            - label: CDKN2A H83Y
+              value: cdkn2a_h83y
+            - label: CDKN2A
+              value: cdkn2a
+        - name: mutation_status_rbm10
+          label: Mutation status/RBM10
+          type: basic:string
+          required: false
+          choices:
+            - label: 'RBM10 F913='
+              value: 'rbm10_f913='
+            - label: 'RBM10 R745*'
+              value: 'rbm10_r745*'
+            - label: RBM10 S846L
+              value: rbm10_s846L
+            - label: 'RBM10 R97*'
+              value: 'rbm10_r97*'
+            - label: RBM10 Q481L
+              value: 'rbm10_q481l'
+            - label: 'RBM10 R295*'
+              value: 'rbm10_r295*'
+            - label: 'RBM10 E732*'
+              value: 'rbm10_e732*'
+            - label: RBM10
+              value: rbm10
+        - name: mutation_status_ptprd
+          label: Mutation status/PTPRD
+          type: basic:string
+          required: false
+          choices:
+            - label: PTPRD S431L
+              value: ptprd_s431l
+            - label: PTPRD
+              value: ptprd
+        - name: mutation_status_u2af1
+          label: Mutation status/U2AF1
+          type: basic:string
+          required: false
+          choices:
+            - label: U2AF1
+              value: u2af1
+        - name: mutation_status_ntrk3
+          label: Mutation status/NTRK3
+          type: basic:string
+          required: false
+          choices:
+            - label: 'NTRK3 F450='
+              value: 'ntrk3_f450='
+            - label: NTRK3 R760H
+              value: ntrk3_r760h
+            - label: NTRK3
+              value: ntrk3
+        - name: mutation_status_smad4
+          label: Mutation status/SMAD4
+          type: basic:string
+          required: false
+          choices:
+            - label: SMAD4 R361H
+              value: smad4_r361h
+            - label: SMAD4 R361C
+              value: smad4_rR361c
+            - label: SMAD4
+              value: smad4
+        - name: mutation_status_pole
+          label: Mutation status/POLE
+          type: basic:string
+          required: false
+          choices:
+            - label: POLE P286R
+              value: pole_p286r
+            - label: POLE V411L
+              value: pole_v411l
+            - label: POLE
+              value: pole
+        - name: mutation_status_lrp1b
```

Comment:
Gene LRP1B has really low expression across many cancers, so it is hard to detect variants using RNA-seq VC in it.